apps/bundle_deploy/bundle_static.c

```diff
@@ -22,7 +22,11 @@
 #include <tvm/runtime/crt/crt.h>
 #include <tvm/runtime/crt/graph_runtime.h>
 #include <tvm/runtime/crt/packed_func.h>
+#include <unistd.h>
```

Comment:
why do we need this header file here?

Reply:
Ah I think I missed this one, it’s probably not needed. But, I won’t be able to address this for a week, so perhaps we can merge as is? Happy to take an issue to fix in a followup PR if that’s ok with you.

Reply:
I suggest this gets trimmed in a follow up PR